R/period_dataset.R

```diff
@@ -408,10 +408,15 @@ derive_vars_period <- function(dataset,
   } else {
     id_vars <- exprs(APHASEN)
   }
-  assert_data_frame(dataset_ref, required_vars = expr_c(subject_keys, new_vars, id_vars))
```

Comment:
I would keep this (and remove `assert_data_frame()` calls in line 359 and 416). At the moment the argument name is not stated in the error message but it should (see https://github.com/pharmaverse/admiraldev/issues/352). Once https://github.com/pharmaverse/admiraldev/issues/352 is implemented the call in line 416 would refer to a dataset unknown by the users.

Reply:
Actioned